Sources/Application/ApplicationMain.swift

```diff
@@ -145,14 +145,36 @@ public func ApplicationMain(_ argc: Int32,
   }
 
   var msg: MSG = MSG()
-  while GetMessageW(&msg, nil, 0, 0) > 0 {
-    TranslateMessage(&msg)
-    DispatchMessageW(&msg)
+
+  let mainRunLoop = RunLoop.current
+
+  var exitCode: Int32 = EXIT_SUCCESS
```

Comment:
For better or worse, the codebase uses the Windows style and Hungarian notation :-(
```suggestion
  var dwExitCode: DWORD = EXIT_SUCCESS
```

Reply:
ApplicationMain returns Int32, and I dont think we should convert msg.wParam (UINT) ot DWORD  and than convert exit code (DWORD aka UInt32) to Int32.
I renamed the variable to nExitCode

Reply:
Ah, agreed.  Thanks.